extensions/wikia/AdEngine/crowdin.conf

```diff
@@ -0,0 +1,4 @@
+[crowdin.project]
+projectid = "ext-AdEngine"
```

Comment:
We can upload existing translations, so if you split it to a new project in the future, you'd just make sure any existing translations for the messages that are moved are uploaded as well and it shouldn't create any more work for translators.